packages/core/src/loaders/http/httpLoader.ts

```diff
@@ -16,7 +16,7 @@ export class HttpLoader {
 
     await this.graphFacade.createRawNode(response.data, {
       type: FilesystemNodeType.File,
-      subtype: trimStart(extname(opts.url), '.'),
+      language: trimStart(extname(opts.url), '.'),
```

Comment:
In Graphite 5 language should be mapped to one of the `Language` enum values.

This is how it's done in Graphite:

```
const extensionToLanguage = {
  md: Languages.Markdown,
  yml: Languages.Yaml,
  js: Languages.JavaScript,
};
```

`extensionToLanguage[ext] || ext`

I suggest to keep a similar map here for the time being. We can centralize it in Graphite later.

Reply:
Dont forget to support yml and yaml :)

Reply:
I'll address this once graphite 5.1 is out